idd/Energy+.idd.in

```diff
@@ -32868,15 +32869,17 @@ ZoneHVAC:PackagedTerminalAirConditioner,
         \type node
         \note Air outlet node for the PTAC must be a zone air inlet node.
    A5,  \field Outdoor Air Mixer Object Type
-        \required-field
         \type choice
         \key OutdoorAir:Mixer
         \note currently only one OutdoorAir:Mixer object type is available.
+        \note This field should be left blank, if the PTAC is connected to central
+        \note dedicated outdoor air through air terminal single duct mixer object.
```

Comment:
1. Nit-picking - there shouldn't be a comma here " . . . left blank, if the . . .".  
2. And I think it would be clearer to say "through an AirTerminal:SingleDuct:Mixer object."
3. More nit-picking, in the existing note "currently only one . . . " please capitalize "Currently".
4. Please add these same comments to the fancoil and watertoairheatpump.